common/async/abort-signals.ts

```diff
@@ -74,3 +74,34 @@ export class BatchedAbortSignals {
     }
   }
 }
+
+/**
```

Comment:
This should be elsewhere, this doesn't really have much to do with abort signals